samples/net/google_iot_device/src/main.c

```diff
@@ -42,9 +45,10 @@ void main(void) {
   if (0 != iotc_example_handle_command_line_args(argc, argv)) {
     return;
   }
+#endif
 
-  if (0 != load_ec_private_key_pem_from_posix_fs(ec_private_key_pem,
-                                                 PRIVATE_KEY_BUFFER_SIZE)) {
+  if (0 != load_ec_private_key_pem(ec_private_key_pem,
```

Comment:
For current state, and sake of simplicity, this function isn't necessary. The `iotc_private_key_pem` could be passed directly to `iotc_connect_private_key_data.crypto_key_union.key_pem.key`.
But if the intention is to show a key-source-ignorant implementation. And/or to prepare for load key from a Secure Element, then this step can stay, doesn't make a huge complexity raise.

Reply:
> But if the intention is to show a key-source-ignorant implementation. 

Yes, I both wanted to minimize changes, and show that the key loading handled by a separate function, so another implementation can be swapped in.

Reply:
👍